src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/RuleBasedTests.cs

```diff
@@ -140,6 +142,24 @@ public async Task CheckPassOfficialValidator(string templateName, string sampleP
             Directory.Delete(resultFolder, true);
         }
 
+        [Theory]
+        [MemberData(nameof(GetHL7V2Cases))]
+        [MemberData(nameof(GetCCDACases))]
+        public async Task CheckPassFhirParser(string templateName, string samplePath)
```

Comment:
Better to add a sample that fails the parsing

Reply:
The original intention is to make sure our conversion result could pass the parser checking. If we add expect-fail-cases, it's more like we are testing the parser itself.
But in order ensure the basic functionality of the parser, I think it's also reasonable to add a basic expect-fail-case, I'll do that later.

Reply:
Yes. Just want to make sure "Bad templates" will fail the test:)